test/e2e/scenario/recorder/shadowDom.scenario.ts

```diff
@@ -80,6 +82,34 @@ const divShadowDom = `<script>
  </script>
  `
 
+/** Will generate the following HTML 
+ * ```html
+ * <my-div id="titi">
+ *  #shadow-root
+ *    <div>toto</div>
+ *</my-div>
+ *```
+ when called like `<my-div />`
+ */
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** it seems to be copy/pasted from previous one‏